extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -68,15 +67,15 @@ class OHIFCornerstoneViewport extends Component {
    * @return {Object} CornerstoneTools Stack
    */
   static getCornerstoneStack(displaySet, dataSource) {
-    const { frameIndex } = displaySet;
+    const { imageIndex } = displaySet;
 
     // Get stack from Stack Manager
     const storedStack = StackManager.findOrCreateStack(displaySet, dataSource);
 
     // Clone the stack here so we don't mutate it
     const stack = Object.assign({}, storedStack);
 
-    stack.currentImageIdIndex = frameIndex;
+    stack.currentImageIdIndex = imageIndex;
```

Comment:
`displaySet.imageIndex` can be `undefined`, but this may break `cornerstone-tools`. You may have to default to `0`?

Reply:
I honestly can't remember